src/hooks/v2v3/LoadV2V3Contract.ts

```diff
@@ -35,8 +35,8 @@ export const useLoadV2V3Contract = ({
       )
 
       setContractJson({
-        address: address ?? contractJson.address,
-        abi: contractJson.abi,
+        address: address ?? contractJson?.address,
```

Comment:
Rinkeby wasn't working locally for me because of this. Maybe good to keep as a safeguard for future?